tests/main/lxd-mount-units/task.yaml

```diff
@@ -23,7 +23,10 @@ execute: |
     fi
 
     # There isn't an official image for mantic yet, let's use the community one
-    if os.query is-ubuntu 23.10; then
+    if os.query is-ubuntu 24.04; then
+        # TODO: remove this once the noble images are published
+        exit 0
+
         CODENAME=$(. /etc/os-release && echo "$VERSION_CODENAME")
         lxc launch --quiet "images:ubuntu/$CODENAME" ubuntu
         core_snap=core22
```

Comment:
Should this comment be updated? `s/mantic/noble`? But it also doesn't seem that `images:ubuntu/$CODENAME` are community images anyway?

Reply:
thanks for the review. I'll fix the mantic/noble.
About the remote images, you can see running `lxd.lxc remote list`

images:ubuntu:<CODENAME> are provided by https://images.linuxcontainers.org
ubuntu:<CODENAME> are provided by https://cloud-images.ubuntu.com

Reply:
I used ubuntu-daily to retrieve the lxd imags instead of the comunity ones when system is ubuntu-24.04

Reply:
Thanks for the explanation!